packages/react/src/components/TabPanel/TabPanel.stories.mdx

```diff
@@ -0,0 +1,52 @@
+import {ArgsTable, Source, Story, Canvas, Meta} from '@storybook/addon-docs';
+import dedent from 'ts-dedent';
+import StoryConfig from '../../../.storybook/story-config.ts';
+import TabPanel from './TabPanel.tsx';
+
+export const meta = {
+  component: TabPanel,
+  title: StoryConfig.TabPanel.hierarchy,
+};
+
+<Meta title={meta.title} component={meta.component} />
+
+export const Template = args => <TabPanel {...args} />;
+
+# TabPanel
+
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+
+## Overview
+
+TabPanel component can be used with Tabs component to implement navigation between different views or sections.
```

Comment:
Shall we have a different `Overview` description? as the `Tab` component's one is similar to this.